triaging.rst

```diff
@@ -43,11 +43,32 @@ or a veteran core developer, they're actively choosing to voluntarily donate the
 time towards the improvement of Python. As is the case with any member of
 the Python Software Foundation, always follow the `PSF Code of Conduct`_.
 
-.. note::
+Becoming a member of Python Triage team
+---------------------------------------
+
+Any Python core developers are welcome to invite a Python contributor to the
+Python Triage team. Do note that the responsibilities of a Python Triage member
+is more elevated than a bug triager in bpo. For example, the Python Triage member
```

Comment:
Another minor nits :) 

This page uses both "bpo" and "b.p.o". Pick one and be consistent.

http://devguide.python.org/tracker/ uses bpo. 

Whilst the target audience may already know what bpo is, consider defining it on first use and consider adding links to bugs.python.org. 

And this is documentation and not some comments in a discussion, so consider writing it out in full as bugs.python.org every time instead of the shorthand.

Reply:
@hugovk I agree that we should use a consistent term for bpo on the page and that a link should be provided. But the usage of the abbreviation is fine, as there is a link for the first mention of the term in each section of the page. If someone is doesn't know what it stands for, they can simply hover over the text to see the url. There wouldn't be too much benefit in fully writing it out each time.